### DISCUSSION TITLE
SMTP settings for email issue

### DISCUSSION THREAD

**Discussion by syeddsherazi** [chatwoot/chatwoot#10580] [Self hosted]
I have setup the self host panel on heroku. And then added the following for smtp:

SMTP_ADDRESS=smtp.gmail.com 
SMTP_PORT=587 
SMTP_DOMAIN=gmail.com 
SMTP_AUTHENTICATION=plain 
SMTP_USERNAME=username
SMTP_PASSWORD=password
MAILER_SENDER_EMAIL=myemail

But I don't believe it's working. Once I create an account on the dashboard panel I don't receive any email for confirmation. Also I keep on resending the email and it doesnt work:
<img width="1440" alt="Screenshot 2024-12-12 at 9 04 19 PM" src="https://github.com/user-attachments/assets/69a4c6c9-47f6-428d-afb0-37e7369c5623" />

If I create any email inbox then also at the end the domain isnt attached. Instead email forwarding format is like: **70d5a1df3389e43b6eb475da452bd4ed@false**

<img width="695" alt="Screenshot 2024-12-12 at 9 05 12 PM" src="https://github.com/user-attachments/assets/0ff460ea-b438-45f7-aea3-0924bd6e6d3f" />

**Comment by sojan-official** [comment]
please ensure conversation continuity is configured : https://www.chatwoot.com/docs/self-hosted/configuration/features/email-channel/conversation-continuity
